demos/pet-demo/src/components/__isograph/.artifact_lookup_cache.json

```diff
@@ -0,0 +1,249 @@
+{
```

Comment:
Hm... IMO we shouldn't write this file to disk. I think it's sufficient to
1. store this in memory
2. recreate the __isograph folder entirely whenever we restart the compiler
3. optimize for watch mode

In the future, I do want to save state between compilations, but 1. it shouldn't live in `__isograph`, but instead somewhere else, and 2. it should be done via something like serde, so that there's no manual maintenance